packages/core/src/components/Price/index.tsx

```diff
@@ -38,7 +38,7 @@ function Price({
   precision = 0,
   round,
   trimTrailingZeros,
-}: PriceProps) {
+}: PriceProps): JSX.Element | null {
```

Comment:
This doesnt seem like it should be necessary?

Reply:
ah, might've been while i was trying something. will double check